CODE_OF_CONDUCT.md

```diff
@@ -1,76 +1,53 @@
-them.
+If at any point concerns are raised that a group or member is not acting according to our values, the behaviour in question should cease, be discussed, and tried to be resolved.
+
+We expect everyone to have a low threshold for raising issues, or in general discuss how we live up to our values. 
+Equally, we encourage all community members to appreciate when concerns are raised and do their best to solve them.
+
+Project maintainers are responsible both for *what* is delivered, as well as *how* it is delivered.
+This includes creating an environment for proper handling of issues raised within the communities.
 
-As well as responsibility for what is delivered the project maintainers are
-also responsible for how we deliver. This includes creating an environment for
-proper handling of issues raised within the communities.
+# Reach out for assistance
 
-# Call out for assistance
-For any problem not directly resolvable within the community, we encourage you
-to call out for assistance. Getting an outsiders perspective on the topic might
-be just what is needed for you to proceed.
+For any problem not directly resolvable within the community, we encourage you to reach out for assistance.
+An outsider’s perspective might be just what is needed for you to proceed.
 
 Send an e-mail to opensource_at_equinor.com and invite for a discussion.
-The email will be handled by a team within the Equinor organization.
+The e-mail will be handled by a team within the Equinor organization.
 
 # Reporting an issue
-If you or your team want to report a direct violation of the code of conduct,
-or a failed attempt in solving a conflict within the community. Contact the
-same team, opensource_at_equinor.com, with a description of the issue.
 
-Your report will be kept confidential from the team or community in question,
-unless you chose to disclose it yourself. We have safe environment for
-reporting issues.
+For reporting direct violations of the code of conduct, or failed attempts at solving conflicts within the community, send a description of the issue to opensource_at_equinor.com.
 
-Consequences of violations are measured after the severity of the issue, or if
-it is a matter of repeatedly failing to take actions with respect to a mandated
-decision.
+To encourage members of the community to file reports, all reports are kept confidential unless you choose to disclose it yourself.
 
-For Equinor maintained projects, only the Equinor open source team has the
-mandate for making a corrective action against individuals based on the code of
-conduct. These decisions are never made within each project alone. As a last
-resort, a potential consequence may be exclusion from participation in that
-particular community.
+Consequences of violations are measured after the severity of the issue, or if it is a matter of repeatedly failing to take actions with respect to a mandated decision.
```

Comment:
Maybe simply "Consequences of violations reflect the severity of the issue, and/or repeated failure to take action following mandated decisions."

Reply:
That looks good!